packages/libs/react-ui/src/components/Input/Input.css.ts

```diff
@@ -68,8 +73,17 @@ export const leadingTextClass = style([
     alignItems: 'center',
     paddingX: '$4',
   }),
+  {
+    minWidth: 'max-content',
+  },
 ]);
 
+export const leadingTextWidthVariant = styleVariants(vars.sizes, (size) => {
+  return {
+    width: size,
```

Comment:
what happens when the text is to big for the width?
should it break the word? use ellipses?

Reply:
example of a large label on smallscreen
![image](https://github.com/kadena-community/kadena.js/assets/4015521/f33eade8-0688-44ea-8910-d944b77d982e)


example of 1 large label with multpile fields
![image](https://github.com/kadena-community/kadena.js/assets/4015521/b13aa5c4-bac0-4ec6-8459-fa60afe80246)

Reply:
i think it should use ellipses